tracing-flame/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "tracing-flame"
+version = "0.1.0"
+authors = ["Jane Lusby <jlusby@yaah.dev>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Before publishing the crate, let's make sure it has complete metadata (we can use other crates in the repo for reference). It would be okay to do that in a follow-up PR, though, as long as it happens before we push anything to crates.io.